src/OpenTelemetry.Exporter.Console/ConsoleActivityExporter.cs

```diff
@@ -71,25 +72,30 @@ public override ExportResult Export(in Batch<Activity> batch)
                             continue;
                         }
 
-                        var array = tag.Value as Array;
-
-                        if (array == null)
+                        if (tag.Value is not Array array)
                         {
                             this.WriteLine($"    {tag.Key}: {tag.Value}");
                             continue;
                         }
 
                         this.WriteLine($"    {tag.Key}: [{string.Join(", ", array.Cast<object>())}]");
                     }
+                }
 
-                    if (!string.IsNullOrEmpty(statusCode))
+                if (activity.Status != ActivityStatusCode.Unset)
+                {
+                    this.WriteLine($"Activity statusCode : {StatusHelper.GetTagValueForActivityStatusCode(activity.Status)}");
+                    if (activity.Status == ActivityStatusCode.Error)
                     {
-                        this.WriteLine($"   StatusCode : {statusCode}");
+                        this.WriteLine($"error : {activity.StatusDescription ?? string.Empty}");
```

Comment:
minor: it seems the upper/lower case is a bit inconsistent in the Console Exporter (e.g. `StatusCode` is CamelCase).